website/src/_posts/2021-12-2.1-2.3.md

```diff
@@ -29,38 +29,38 @@ Uppy now has a new `@uppy/audio` plugin for recording audio memos, sounds, Jingl
 
 ## `@uppy/unsplash` is now production ready
 
-The Unsplash plugin has received an update, it provides attribution to the image authors, and adds some polish to make it look better. Being fully brand compliant with Unsplash means it’s now production-ready. You can demo the plugin on the [website](https://uppy.io/examples/dashboard) or read the [docs](https://uppy.io/docs/unsplash/).
+The Unsplash plugin has received an update, providing attribution to the image authors and adding some polish to make it look better. Being fully brand compliant with Unsplash means it’s now production ready. You can demo the plugin on the [website](https://uppy.io/examples/dashboard) or read the [docs](https://uppy.io/docs/unsplash/).
 
 ## New streaming interface for Companion
 
-[Companion][companion] handles the server-to-server communication between your server and file storage providers such as Google Drive, Dropbox, Instagram, and the like. This bypasses the client (so a 5 GB video isn’t eating into your users’ data plans) and is directly uploaded to the final destination. Companion is made to be effortlessly integrate-able with Uppy.
+[Companion][companion] handles the server-to-server communication between your server and file storage providers such as Google Drive, Dropbox, Instagram, and the like. This bypasses the client (so a 5 GB video isn’t eating into your users’ data plans) and files are directly uploaded to the final destination. Companion is made to be effortlessly integrate-able with Uppy.
 
-It works by downloading the requested file from a remote provider on the server to then start uploading it to the destination. This does the job but it’s not ideal.
+It works by downloading the requested file from a remote provider on the server to then start uploading it to the destination. This does the job, but it’s not ideal.
```

Comment:
This sentence still confused me a bit. Shouldn't this say: 'It use to work by..."? Or is this not a description of the functionality before the update?

It's also not completely clear to me what 'it' refers to. I suppose 'Companion'?

Reply:
Yes, good point about “it used to”.